### DISCUSSION TITLE
Windows antivirus didn't like the newest CLI (1.10.3)

### DISCUSSION THREAD

**Discussion by mortensimon** [stripe/stripe-cli#888] [General]
Not sure if it's me or not, but I got this when I tried to download the newest Stripe CLI for Windows:
![image](https://user-images.githubusercontent.com/5700039/172026782-433057c7-9b0d-4fd9-9566-acb7b0a52da7.png)

**Comment by Rikki-Tavi** [comment]
I just had similar experience with Norton AV removing stripe.exe as soon as I unzipped it from same zip file. By comparison, the reason I was given is mild compared to what mortensimon received from his AV protection. But now I'm afraid to try this technology. Norton screen shots follow:
![image](https://user-images.githubusercontent.com/18092994/172070293-82c31366-e63e-4c26-b085-7e21711c32a1.png)

![image](https://user-images.githubusercontent.com/18092994/172070325-779549e4-3066-488a-b5c0-e056aca14bba.png)

**Reply by Rikki-Tavi** [reply]
I uploaded the zip file to VirusTotal and it came out clean. So I decided to move forward with it.
https://www.virustotal.com/gui/file/881bd1131cf0d3b3c8f701a96cc15cafadfb08c9b8c0cc6fe2d6077f4b015889
